### PR TITLE
UIDATIMP-176: implement delete match profile feature

### PR DESCRIPTION
**Pull request by OleksandrYehorovEPAM**

### CODE REVIEW

File: src/settings/SettingPage/getCRUDActions.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        layer: null,
+      });
+
+      onDeleteSuccess(record);
+    } catch (error) {
+      onDeleteError(record, error);
+    }
+  };
+
+  // const deleteMatchProfile = async record => {
```
Comment (VictorSoroka1) [inline src/settings/SettingPage/getCRUDActions.js]: Will commented code to be changed/removed?

File: src/settings/SettingPage/getCRUDActions.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    });
+
+  const editRecord = record => resourceMutator.PUT(record)
+    .catch(error => {
+      onUpdateRecordError(record, error);
+
+      throw error;
+    });
+
+  // const showDeleteRecordSuccessfulMessage = record => {
```
Comment (VictorSoroka1) [inline src/settings/SettingPage/getCRUDActions.js]: Will commented code to be changed/removed?

File: src/settings/SettingPage/getCRUDActions.js (hunk 1 above)
Comment (OleksandrYehorovEPAM) [reply src/settings/SettingPage/getCRUDActions.js]: Removed

File: src/settings/SettingPage/getCRUDActions.js (hunk 2 above)
Comment (OleksandrYehorovEPAM) [reply src/settings/SettingPage/getCRUDActions.js]: Removed

File: src/settings/SettingPage/utils.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
```
Comment (VictorSoroka1) [inline src/settings/SettingPage/utils.js]: Add an empty line.

**Pull request merged**